spanner/google/cloud/spanner_v1/database.py

```diff
@@ -406,6 +438,258 @@ def __exit__(self, exc_type, exc_val, exc_tb):
         self._database._pool.put(self._session)
 
 
+class BatchTransaction(object):
```

Comment:
Users should be able to do normal read/queries also on BatchTransaction. Can they do that?

Reply:
Added in 5342ccd